providers/bind/soa.go

```diff
@@ -19,10 +23,22 @@ func makeSoa(origin string, defSoa *SoaDefaults, existing, desired *models.Recor
 		desired = &models.RecordConfig{}
 	}
 
+	soaMail := firstNonNull(desired.SoaMbox, existing.SoaMbox, defSoa.Mbox, "DEFAULT_NOT_SET.")
+	if strings.Contains(soaMail, "@") {
+		res := strings.SplitN(soaMail, "@", 2)
+		user_part, domain_part := res[0], res[1]
+		// RFC-1035 [Section-8]
+		user_part = strings.ReplaceAll(user_part, ".", "\\.")
+		soaMail = user_part + "." + domain_part
+	} else {
```

Comment:
change `Note:` to `WARNING:` (in both places)

Reply:
Done